src/phylum/ci/cli.py

```diff
@@ -149,8 +149,8 @@ def get_args(args: Optional[Sequence[str]] = None) -> tuple[argparse.Namespace,
         "-f",
         "--force-analysis",
         action="store_true",
-        help="""Specify this flag to force analysis, even when no dependency file has changed. This flag is implicitly
-            set when *any* manifest is included, to maximize the chance that updated dependencies are accounted.""",
+        help="""Specify this flag to force analysis, even when no dependency file has changed. Implicitly set for newly
+            created projects or when *any* manifest is included, to account for possible dependency updates.""",
```

Comment:
Honestly I don't think this second sentence is necessary, considering that most people would assume the delta is compared to the last run which is nothing in this case.

Otoh, it probably doesn't hurt.

Reply:
Yeah, I deliberated over the wording here. I wanted to keep it short (and it is shorter than prior to this PR) but still make it clear which instances the behavior takes effect. If nothing else, I use the `--help` output to remind myself of the behavior/interactions and having it like this is beneficial to me. Hopefully, it will provide value to other users as well.